tests/system/model_monitoring/assets/models.py

```diff
@@ -83,23 +83,13 @@ async def predict_async(self, body):
 
 
 class MyRemoteModel(mlrun.serving.states.Model):
-    def __init__(self, name, raise_exception, artifact_uri, **kwargs):
-        super().__init__(
-            name=name,
-            raise_exception=raise_exception,
-            artifact_uri=artifact_uri,
-            **kwargs,
-        )
-        self.artifact = None
+    execution_mechanism = "naive"
```

Comment:
execution_mechanism is no longer a class attribute, it is a prameter of add_model